README.md

```diff
@@ -20,7 +20,7 @@ The App Center SDK uses a modular architecture so you can use any or all of the
 
 5. **App Center Data**: The App Center Data service provides functionality enabling developers to persist app data in the cloud in both online and offline scenarios. This enables you to store and manage both user-specific data as well as data shared between users and across platforms. In addition, this service includes basic conflict resolution out of the box. **Not available for macOS*.
 
-6. **App Center Identity**: The App Center Identity is an identity management service that allows you to customize and control how your users interact with your app. We provide an out of the box UI that allows app developers to authenticate their users to an app. In addition, app developers may use and track the user's identity across App Center services. **Not available for macOS*.
+6. **App Center Auth**: The App Center Auth is an identity management service that allows you to customize and control how your users interact with your app. We provide an out of the box UI that allows app developers to authenticate their users to an app. In addition, app developers may use and track the user's identity across App Center services. **Not available for macOS*.
```

Comment:
Maybe:
```suggestion
6. **App Center Auth**: The App Center Auth is an authentication management service that allows you to customize and control how your users interact with your app. We provide an out of the box UI that allows app developers to authenticate their users to an app. In addition, app developers may use and track the user's identity across App Center services. **Not available for macOS*.
```

Reply:
```suggestion
**App Center Auth**: The App Center Auth service is an authentication management service that allows you to customize and control how your users interact with your app. We provide an out of the box UI that allows app developers to authenticate their users to an app. In addition, app developers may use and track the user's identity across App Center services. **Not available for macOS*.
```

Reply:
The sentence is odd without the missing service word otherwise.

Reply:
I would like to get a PM confirmation for the changes before taking it.

Reply:
Final version
```suggestion
6. **App Center Auth**: The App Center Auth service allows you to customize and control how your users interact with your app. We provide an out of the box UI that allows app developers to authenticate their users to an app. In addition, app developers may use and track the user's identity across App Center services. **Not available for macOS*.
```